screens/shopsearchscreen.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image, Dimensions } from 'react-native';
+
+
+export default class Shopsearchscreen extends React.Component {
```

Comment:
Use Pascal Case - it's just like camel case but the first word is also uppercase - it should be ShopSearchScreen